### PR TITLE
Use the generated models from the operator

### PR DESCRIPTION
**Pull request by andreaTP**
Demo PR to show-case https://github.com/Apicurio/apicurio-registry-operator/pull/187

### CODE REVIEW

File: system-tests/pom.xml [hunk 1]
```diff
@@ -209,7 +224,45 @@
...
+                <version>1.6.1</version>
+                <executions>
+                    <execution>
+                    <!-- Needs to run before java-generator-maven-plugin -->
+                    <phase>generate-sources</phase>
+                    <goals>
+                        <goal>wget</goal>
+                    </goals>
+                    <configuration>
+                        <url>https://raw.githubusercontent.com/Apicurio/apicurio-registry-operator/1.0.0-v2.0.0.final/packagemanifests/1.0.0-v2.0.0.final/registry.apicur.io_apicurioregistries.yaml</url>
```
Comment (andreaTP) [inline system-tests/pom.xml:240]: @jsenko this is the latest "release" I found, do you think that the tag is accurate?

**Comment by andreaTP** [comment]
The `native-image` pipeline should be fixed by #2777 2777

**Comment by andreaTP** [comment]
I apparently broke some tests 😅 looking into it on Mon 🙂

File: system-tests/src/main/java/io/apicurio/registry/systemtests/framework/ApicurioRegistryUtils.java [hunk 2]
```diff
@@ -18,7 +17,7 @@ public class ApicurioRegistryUtils {
     private static final Logger LOGGER = LoggerUtils.getLogger();
 
     private static String getTruststoreSecretName(ApicurioRegistry registry) {
-        ApicurioRegistrySpecConfigurationKafkaSecurity security = registry
+        io.apicur.registry.v1.apicurioregistryspec.configuration.kafkasql.Security security = registry
```
[the remark this answers is not in the document]
Comment (obabec) [reply system-tests/src/main/java/io/apicurio/registry/systemtests/framework/ApicurioRegistryUtils.java:20]: +1

**Review by obabec — APPROVED** [review]
I think it is great! I would only suggest removing the full path class selector. Otherwise great. Thank you @andreaTP !

**Pull request merged**